rfc/rfc-0010-drop-core-runtime.md

```diff
@@ -0,0 +1,37 @@
+# RFC-#: Drop Core Runtime
+
+**Authors:** @scothis
+
+**Status:**
+
+**Pull Request URL:** [#1373](https://github.com/projectriff/riff/pull/1373)
+
+**Superseded by:** N/A
+
+**Supersedes:** N/A
+
+**Related:** N/A
+
+## Problem
+
+riff currently has two ways to deploy HTTP triggered workloads: the Core and Knative runtimes. This dichotomy causes users extra cognitive load. For every operation they have to think about which runtime they are targeting. Moreover, the capabilities and behavior of each runtime are subtly different.
+
+As the Knative runtime is a super-set of the Core runtime, we should consolidate on the Knative runtime. Users who don't want to run Knative, can take a riff built container image and deploy it however they like.
+
```

Comment:
Should we then also rename the `knative` command to something like `serving` or `http` to avoid locking in to the specific name of the implementation used?

Reply:
I've had the same thought, but is somewhat orthogonal to the decision here

Reply:
Maybe another RFC ;)